docs/static_landing_page/assets/vendor/bootstrap/css/bootstrap-grid.css

```diff
@@ -0,0 +1,3872 @@
+/*!
```

Comment:
I'm assuming you didn't write this file and some others. Can we pull them in at build time rather than checking them into the code base?

Reply:
Yes, you can do that but except this file: assets/vendor/icofont/icofont.min.css .